drivers/soundwire/intel_init.c

```diff
@@ -30,6 +30,48 @@ static int ctrl_link_mask;
 module_param_named(sdw_link_mask, ctrl_link_mask, int, 0444);
 MODULE_PARM_DESC(sdw_link_mask, "Intel link mask (one bit per link)");
 
+static void intel_link_auxdev_release(struct device *dev)
+{
+	struct auxiliary_device *auxdev = to_auxiliary_dev(dev);
+	struct sdw_intel_link_dev *ldev = container_of(auxdev,
+						       struct sdw_intel_link_dev,
+						       auxdev);
```

Comment:
can we use the macro auxiliary_dev_to_sdw_intel_link_dev() here?

Reply:
same here, I didn't remember defining this macro... I'll blame it on rotating outages